sites/public/cypress/mockData/applicationData.ts

```diff
@@ -307,6 +277,107 @@ export const ElmVillageApplication: Application = {
   language: LanguagesEnum.en,
 }
 
+const BlueSkyProgramsAndPreferences = {
+  preferences: [],
+  programs: [
```

Comment:
suggestion: I don't think programs are needed in BlueSky so this array could probably be changed to an empty array

Reply:
Removing this caused the same issue for me on the household changes step so I fixed that too